fades/main.py

```diff
@@ -299,7 +299,7 @@ def go():
 
     if create_venv:
         # Check if the requested packages exists in pypi.
-        if not args.no_precheck_availability:
+        if not args.no_precheck_availability and indicated_deps:
```

Comment:
this will pass and still give the message incorrectly if has not-pypi dependencies... you should check kind of doing...
```suggestion
        if not args.no_precheck_availability and indicated_deps.get('pypi'):
```